modules/ai/assets/js/media-library/generate.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { RequestIdsProvider } from '../editor/context/requests-ids';
+import styled from 'styled-components';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import App from '../editor/app';
+
+const AIMedia = ( { onClose } ) => {
+	return (
+		<>
+			<App
+				type={ 'media' }
+				getControlValue={ () => {} }
+				setControlValue={ () => {} }
+				onClose={ onClose }
+			/>
+		</>
+	);
+};
+
+AIMedia.propTypes = {
+	onClose: PropTypes.func.isRequired,
+};
+
+const Icon = styled.i`
+	padding-right: 0.5em;
+	cursor: pointer;
+`;
+
+const StyledButton = styled.a`
+	color: #C00BB9;
+	cursor: pointer;
+	font: inherit;
+	display: inline-block;
+	position: relative;
+	top: -3px;
+	margin-left: 4px;
+	border: 1px solid #C00BB9;
+	border-radius: 3px;
+	line-height: 2.15384615;
+	padding: 0 10px;
+	background: #f6f7f7;
+
+	&:hover {
+		color: #C00BB9;
+		background: #eeefef;
+	}
+`;
```

Comment:
Can it be a reusable component?

Reply:
Sure, but not sure we'll need to since each WP screen has its own design and we adjust ourselves accordingly when we insert a new element. If we'll see a need for future additions we can easily take it out to a shared utils.